src/core/Table/Table.tsx

```diff
@@ -88,6 +92,22 @@ export type TableProps<
    * @default 300
    */
   intersectionMargin?: number;
+  /**
+   * A function that will be used for rendering a component for each row if that row is expanded.
+   * Component will be placed right after the row, in the same row group div.
+   */
+  subComponent?: (row: Row<T>) => React.ReactNode;
+  /**
+   * Handler for row expand events. Will trigger when expanding and condensing rows.
+   */
+  onExpand?: (
+    selectedData: T[] | undefined,
```

Comment:
Maybe not `selected` but `expandedData`.

Reply:
Changed